### PR TITLE
Correct ihMTsat

### PR DESCRIPTION
**Pull request by karanphil**
Here is a small modification to the computation of ihMTsat. Previously, it was in fact the ihMTdR1sat that was calculated. To output this metric, use the `--legacy_sat` option. If not, the real ihMTsat will be computed.

@Manonedde

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_compute_ihMT_maps.py [hunk 1]
```diff
@@ -110,6 +110,8 @@ def _build_arg_parser():
                         'Not recommended.')
     p.add_argument('--single_echo', action='store_true',
                    help='Use this option when there is only one echo.')
+    p.add_argument('--legacy_sat', action='store_true',
+                   help='Use this option to choose the ihMTdR1sat contrast.')
```
Comment (Manonedde) [inline scripts/scil_compute_ihMT_maps.py:114]: I will add a "Not recommended" in the help of the --legacy_sat option.

**Review by Manonedde — CHANGES_REQUESTED** [review]

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**